modules/installation/pages/install.adoc

```diff
@@ -191,6 +192,7 @@ Here is a description of all the fields in the config file:
   *** `"AppRoot"`: Absolute path to where application folder will be.
   *** ``"DataRoot``": Absolute path to where the data folder will be.
   *** `"LogRoot"`: Absolute path to where the log folder will be.
+**** For production systems, it’s recommended that you use a separate volume or logical partition to store log files to prevent out-of-space issues for critical components.
   *** `"TempRoot"`: Absolute path to where the temp folder will be.
```

Comment:
**** For production systems, it’s recommended that you use a separate disk partition to store log files to prevent out-of-space issues.